LiskHQ/lisk-desktop
# Bump lisk client version - Closes #4080, #4076

### What was the problem?

This PR resolves #4080, #4076

### How was it solved?

- Bumping version

### How was it tested?

- npm run dev
- Ensure the issue #4076 is resolved
- Ensure the multi-signature registration and transactions work for regular and hardware wallet accounts
